lib/streams.gi

```diff
@@ -1084,14 +1084,33 @@ fi;
 #M  OutputTextFile( <str>, <append> )
 ##
 InstallMethod( OutputTextFile,
-    "output text stream from file",
+    "output text stream to file",
     [ IsString,
       IsBool ],
 function( str, append )
     local   fid;
     str := UserHomeExpand(str);
 
-    fid := OUTPUT_TEXT_FILE( str, append );
+    fid := OUTPUT_TEXT_FILE( str, append, false );
+    if fid = fail  then
+        return fail;
+    else
+        atomic OutputTextFileStillOpen do
+            AddSet( OutputTextFileStillOpen, fid );
+        od;
+        return Objectify( OutputTextFileType, [fid, Immutable(str), true] );
+    fi;
+end );
+
+InstallMethod( OutputGzipFile,
```

Comment:
Could you perhaps move this below the second `OutputTextFile` and add a comment like we use for other stuff? ```suggestion
#############################################################################
##
 #M  OutputGzipFile( <str>, <append> )
 ##
InstallMethod( OutputGzipFile,
```
This would preserve grouping, and also the diff here is easier to read. I am trying to understand why the CI tests fail and it is difficult to grok what happens.

Alternatively, I'd be happy to make that change, I just don't want to make it "behind your back"